### PR TITLE
Add AGC with Julia project

### PR DESCRIPTION
**Pull request by Moelf**
@alexander-held let me know if you want split into 2 projects

### CODE REVIEW

File: projects/agc-julia-rntuple.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Julia for the Analysis Grand Challenge
+postdate: 2023-04-07
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+experiments:
```
Comment (alexander-held) [inline projects/agc-julia-rntuple.yml:9]: Let's add ATLAS here too, the pieces are generic enough that this broadly applies to ATLAS as well.

**Comment by alexander-held** [comment]
Thanks for preparing this! I think having the RNTuple as an optional piece of this works, otherwise with two projects we probably wouldn't be able to have mentoring for both? This leaves flexibility, which I think is good.

**Comment by Moelf** [comment]
`build` CI failed but I don't see why that value is "unexpected"

**Comment by davidlange6** [comment]
"Julia" wasn't in https://github.com/research-software-collaborations/project_database/blob/main/project_metadata.yml#L24 until I added it

**Pull request merged**